### PR TITLE
API-2908: update category page to use route hooks

### PR DESCRIPTION
**Pull request by TS000**
### Description
Replaced React Router 4 prop based params with React Router 5 useParams 🎸 

- [ n/a ] Tests added or updated for change
- [ n/a ] Docs updated
- [ n/a ] Accessibility concerns have been considered and addressed

### Requested feedback
All feedback appreciated.

### CODE REVIEW

File: src/containers/documentation/AuthorizationDocs.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable prefer-arrow/prefer-arrow-functions */
```
Comment (anthony-kopczyk) [inline src/containers/documentation/AuthorizationDocs.tsx:1]: I haven't looked at the entire file. Is this still needed?

File: src/containers/documentation/CategoryPage.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
-import { RouteComponentProps } from 'react-router';
+import { Flag } from 'flag';
```
Comment (anthony-kopczyk) [inline src/containers/documentation/CategoryPage.tsx:2]: There's a duplicate `import { Flag }` but from a different file on line 9. I think line 9 is correct and this one can be removed but not certain.

**Review by anthony-kopczyk — CHANGES_REQUESTED** [review]
1. I think maybe this was based off of the branch for updating authorizationDocs. Shouldn't be a big deal because that PR is ready to be merged already. But probably doin't want to merge this until that one is merged first
2. There's a duplicate import it looks like in `CategoryPage`
3. There's a line in `AuthorizationDocs` that might be able to be removed? Doesn't need to be in this PR though.

File: src/containers/documentation/AuthorizationDocs.tsx (hunk 1 above)
Comment (TS000) [reply src/containers/documentation/AuthorizationDocs.tsx:1]: I'm confused as to why this file was part of my PR.

**Review by anthony-kopczyk — APPROVED** [review]
LGTM

**Pull request merged**